fixed tab navigation and cleaned code

KatarzynaStefanowiczPagepro opened the pull request:
## Description

- [] TODO write description

**New stuff** ✨

* New stuff goes here

**Changes** 🏗

* Changes go here

**Deletions** ⚰️

* Deletions go here

## TODO

- [ ] Add TODOs

(Resolves | Contributes to) #31415

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/JoinColony/colonyCDapp?pullRequest=662) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/JoinColony/colonyCDapp?pullRequest=662) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/JoinColony/colonyCDapp?pullRequest=662) it.</sub>

KatarzynaStefanowiczPagepro commented:
https://tw.auroracreation.com/app/tasks/15439879
Fixed navigating tabs on mobile (bug: https://jam.dev/c/a6569745-0462-4be3-9b95-fb5b156c8129)
Cleaned code: created heading-5 and text-1 classes in tailwind to avoid using repeatable styles
